install.ps1

```diff
@@ -39,16 +39,54 @@ $repo = "onflow/flow-cli"
 $versionURL = "https://api.github.com/repos/$repo/releases/latest"
 $assetsURL = "https://github.com/$repo/releases/download"
 
-# Add the GitHub token to the web request headers if it was provided
-$webRequestOptions = if ($githubToken) {
-	@{ 'Headers' = @{ 'Authorization' = "Bearer $githubToken" } }
-} else {
-    @{}
+# Function to get the latest version
+function Get-Version {
+    param (
+        [string]$repo,
+        [string]$searchTerm,
+        [string]$githubTokenHeader
+    )
+
+    $page = 1
+    $version = null
+
+    while (-not $version) {
+        if ($githubTokenHeader) {
+            $response = Invoke-WebRequest -Uri "https://api.github.com/repos/$repo/releases?per_page=100&page=$page" -Headers @{ 'Authorization' = $githubTokenHeader } -UseBasicParsing -ErrorAction SilentlyContinue
```

Comment:
Just confirming we want `-ErrorAction SilentlyContinue` here

Reply:
Yup we need it so that it doesn't throw an exception when it doesn't receive an OK response.  We catch these manually and decide whether to continue iterating pages.